SDL/Quintuplet.cu

```diff
@@ -428,6 +428,117 @@ __device__ bool SDL::runQuintupletDefaultAlgo(struct SDL::modules& modulesInGPU,
+        mdsInGPU.anchorPhi[firstMDIndex],                             // inner (hit 1) t3_0_phi
+        mdsInGPU.anchorZ[firstMDIndex],                               // inner (hit 1) t3_0_z
+        sqrtf(x1*x1 + y1*y1),                                         // inner (hit 1) t3_0_r
+        float(modulesInGPU.layers[lowerModuleIndex1] + 6*is_endcap1), // inner (hit 1) t3_0_layer
+        mdsInGPU.anchorEta[secondMDIndex],                            // inner (hit 2) t3_2_eta
+        mdsInGPU.anchorPhi[secondMDIndex],                            // inner (hit 2) t3_2_phi
+        mdsInGPU.anchorZ[secondMDIndex],                              // inner (hit 2) t3_2_z
+        sqrtf(x2*x2 + y2*y2),                                         // inner (hit 2) t3_2_r
+        float(modulesInGPU.layers[lowerModuleIndex2] + 6*is_endcap2), // inner (hit 2) t3_2_layer
+        mdsInGPU.anchorEta[thirdMDIndex],                             // inner (hit 3) t3_4_eta
+        mdsInGPU.anchorPhi[thirdMDIndex],                             // inner (hit 3) t3_4_phi
+        mdsInGPU.anchorZ[thirdMDIndex],                               // inner (hit 3) t3_4_z
+        sqrtf(x3*x3 + y3*y3),                                         // inner (hit 3) t3_4_r
+        float(modulesInGPU.layers[lowerModuleIndex3] + 6*is_endcap3), // inner (hit 3) t3_4_layer
+        log10(2.99792458e-3f*3.8f*outerRadius),                       // outer t3_pt
+        mdsInGPU.anchorEta[thirdMDIndex],                             // outer (hit 4) t3_0_eta
+        mdsInGPU.anchorPhi[thirdMDIndex],                             // outer (hit 4) t3_0_phi
+        mdsInGPU.anchorZ[thirdMDIndex],                               // outer (hit 3) t3_0_z
+        sqrtf(x3*x3 + y3*y3),                                         // outer (hit 3) t3_0_r
+        float(modulesInGPU.layers[lowerModuleIndex3] + 6*is_endcap3), // outer (hit 3) t3_0_layer
+        mdsInGPU.anchorEta[fourthMDIndex],                            // outer (hit 4) t3_2_eta
+        mdsInGPU.anchorPhi[fourthMDIndex],                            // outer (hit 4) t3_2_phi
+        mdsInGPU.anchorZ[fourthMDIndex],                              // outer (hit 4) t3_2_z
+        sqrtf(x4*x4 + y4*y4),                                         // outer (hit 4) t3_2_r
+        float(modulesInGPU.layers[lowerModuleIndex4] + 6*is_endcap4), // outer (hit 4) t3_2_layer
+        mdsInGPU.anchorEta[fifthMDIndex],                             // outer (hit 5) t3_4_eta
+        mdsInGPU.anchorPhi[fifthMDIndex],                             // outer (hit 5) t3_4_phi
+        mdsInGPU.anchorZ[fifthMDIndex],                               // outer (hit 5) t3_4_z
+        sqrtf(x5*x5 + y5*y5),                                         // outer (hit 5) t3_4_r
+        float(modulesInGPU.layers[lowerModuleIndex5] + 6*is_endcap5), // outer (hit 5) t3_4_layer
+        log10((innerRadius+outerRadius)*3.8f*1.602f/(2*100*5.39f)),   // t5_pt
```

Comment:
What are those numbers? If they are constants, could they be added in the proper file and included here? If they are some temporary, magic numbers, could they be added in this file with a descriptive name (and possibly a descriptive comment), so that one can follow the calculations?